api/src/pcapi/core/educational/validation.py

```diff
@@ -98,7 +98,9 @@ def check_confirmation_limit_date_has_not_passed(booking: models.CollectiveBooki
 
 def check_collective_stock_is_editable(stock: models.CollectiveStock) -> None:
     offers_validation.check_validation_status(stock.collectiveOffer)
-    offers_validation.check_event_expiration(stock)
+    for booking in stock.collectiveBookings:
+        if booking.status is not models.CollectiveBookingStatus.CANCELLED:
+            offers_validation.check_event_expiration(stock)
```

Comment:
```suggestion
    for booking in stock.collectiveBookings:
        if booking.status is not models.CollectiveBookingStatus.CANCELLED:
            offers_validation.check_event_expiration(stock)
            break
```

D'un point de vue métier il ne peut pas y avoir plusieurs booking non cancelled pour un stock mis je préfère être sure.